.github/ISSUE_TEMPLATE/engineering-manager-onboarding.md

```diff
@@ -0,0 +1,67 @@
+
+     It will be your task to improve [this document](/.github/ISSUE_TEMPLATE/engineering-manager-onboarding.md), drawing from all the things you’ve had to learn. You’re not on your own, your fellow peers will all be available to support you during this task. You got this!
+
+- [ ] The people-ops team should have signed you up for the various on-site manager trainings.
+
+- [ ] Your manager should have invited you to meetings relevant to managers.
+
+- [ ] Your manager should have scheduled 1:1s for you with the other engineering managers, including the CTO, and other leadership people.
+
+## Your first ~two months
+
+The goal of onboarding is to set you up for success as a new engineering manager at Artsy. To that end, your main objective during this time will be to gather knowledge. You will be given ample time to get acquainted with your new peers, our business, and our tech.
+
+_For now we’ll try to target roughly two months, as there are currently 5 product teams and sprints have a duration of 2 weeks. Whether or not this duration was too arbitrary is something we’ll have to learn._
+
+To help work on the main objective, we’ve tried to identify the following goals.
+
+### Get to know Artsy's team
+
+- [ ] Have 1:1s with each and every engineer (that includes tech-leads) to get to know the people, their interests, their north-star goals, and their perspectives on the businesses they contribute to.
+
+- [ ] While working with individual contributors and during your 1:1s, try to develop an understanding of where and how you want to add value as a manager.
+
+- [ ] Keep a continuous and open discussion going with your peer managers about opportunities for people you think you could manage and why. By the end of sprint rotation we should organically have arrived at a set of suggestions that we, as the entire engineering team, can _consider_.
+
+### Get to know Artsy's tech
+
+- [ ] Unlike individual contributors, who rotate teams for a few sprints to gain a general understanding of how we work as contributors, engineering managers are expected to rotate across _all_ the teams to gain the understanding needed to successfully support their reports in achieving their goals and growth. (For example, unblocking people by knowing how to achieve something or who/where to delegate to, facilitate discussions by understanding context and who to involve for a given topic; and how to set expectations for engineers, product, and business owners as to what is realistically achievable.) Tech-leads and PMs should have been made aware of your main objective and provide you with time to do so.
```

Comment:
Maybe we can move "engineering managers are expected to rotate across _all_ the teams" expectation to the top of this "Your first ~two months" section, given it's main format to know Artsy's team, tech, and business.

Reply:
I hear what you’re saying and also had it like that in an earlier iteration, but seeing as we’re calling out opportunities to meet with the team and business people separately, I think it’s ok to scope this to the tech aspect. If you or others feel more strongly about this then let’s revisit.